qml/Stage/ApplicationWindow.qml

```diff
@@ -119,6 +121,12 @@ FocusScope {
                 focusedSurface.focus = true;
             }
         }
+
+        onLiveSurfaceChanged: {
+            if (liveSurface) {
+                surfaceSeenLive = true;
```

Comment:
nitpick: I would do this on line 74 in the Connections element, to save one signal connection.